.github/workflows/changeset.yaml

```diff
@@ -2,7 +2,7 @@ name: Changesets
 on:
     push:
         branches:
-            - releases
+            - release
```

Comment:
The branch name is releases .. https://github.com/DeuxHuitHuit/288-toolkit/tree/releases

Reply:
Can it change? Or should we just adjust the docs instead?